gatsby-config.js

```diff
@@ -4,11 +4,21 @@ let CONSTS = require("./config/gatsby-config-consts");
 if (!CONSTS) CONSTS = {};
 
 const buildMode = process.env.BUILD_ENV || "production";
-const siteUrl =
-	process.env.SITE_URL ||
-	(buildMode === "production"
-		? "https://unicorn-utterances.com"
-		: "https://beta.unicorn-utterances.com");
+
+let siteUrl = process.env.SITE_URL;
+
+if (!siteUrl) {
+	switch (buildMode) {
+		case "production":
+			siteUrl = "https://unicorn-utterances.com";
+			break;
+		case "local":
+			siteUrl = "localhost:8000";
+			break;
+		default:
+			siteUrl = "https://beta.unicorn-utterances.com";
+	}
+}
```

Comment:
This should keep previous behavior, but is one change made to a deploy related variable